react-select/react-select.d.ts

```diff
@@ -1,80 +1,226 @@
-// Type definitions for react-select v0.9.10
+// Type definitions for react-select v1.0.0
 // Project: https://github.com/JedWatson/react-select
-// Definitions by: ESQUIBET Hugo <https://github.com/Hesquibet/>
+// Definitions by: Gilad Gray <https://github.com/giladgray/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-///<reference path="../react/react.d.ts"/>
+/// <reference path="../react/react.d.ts"/>
 
 // Typings for https://github.com/JedWatson/react-select
-//***Usage***
-// import ReactSelect = require('react-select');
+// import * as Select from "react-select";
```

Comment:
I don't think this comment block needs to be here; the tests and docs are a better place